packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -31,6 +31,7 @@ export const getStyles = memoizeFunction((
         height: 'auto',
         boxSizing: 'border-box',
         border: '1px solid transparent',
+        borderRadius: 0,
```

Comment:
What's with this change?

Reply:
Oh important note!!! On Mac for both Safari and Chrome, button elements now default to 2px borderRadius! So suddenly all of our buttons are getting borderRadius everywhere. You can see it all over our products. This one change needs to get in sooner than later, and i probably shouldn't have put it in this PR.

I'll pull it out into a separate one.